factory/mock/components/components.go

```diff
@@ -0,0 +1,866 @@
+package components
```

Comment:
wondering if we could move these in `testscommon/components`

Reply:
:+1:

Reply:
moved